### PR TITLE
ClubPage Styling

### PR DESCRIPTION
**Pull request by steven-solar**
This PR styles the assignments and comments on the ClubPage, and creates a Mark as Done button (functionality to be implemented later, issue #90). 

Screenshot 2020-07-20 at 9 03 06 PM

### CODE REVIEW

File: capstone-frontend/src/styles/Groups.css [hunk 1]
```diff
@@ -2,6 +2,40 @@
...
+  min-width: fit-content;
+  height: fit-content;
+}
+
+.assignment-text {
+  font-size: 22px;
+  font-weight: 500;
+}
+
 .assignment-border {
```
Comment (antoniolinhart) [inline capstone-frontend/src/styles/Groups.css]: nit: alphabetize styles

File: capstone-frontend/src/styles/Groups.css [hunk 2]
```diff
@@ -2,6 +2,40 @@
   margin-top: 20px;
 }
 
+.assignment-card {
+  text-align: left;
+  border: 1px solid grey;
+  box-shadow: 3px 3px 3px #888888;
+  margin-bottom: 10px;
```
Comment (antoniolinhart) [inline capstone-frontend/src/styles/Groups.css]: nit: alphabetize specific styles

File: capstone-frontend/src/styles/Groups.css [hunk 3]
```diff
@@ -33,6 +67,36 @@
   text-decoration: none;
 }
 
+.comment-card {
+  display: flex;
+  align-items: center;
+  border-top: 0.1px solid grey;
+  background-color: #f5f5f5;
+  padding: 15px 10px;
```
Comment (antoniolinhart) [inline capstone-frontend/src/styles/Groups.css:54]: Same as above

**Review by antoniolinhart — APPROVED** [review]
I really like the new styling on this page! Only a few small changes to make

File: capstone-frontend/src/styles/Groups.css (hunk 1 above)
Comment (steven-solar) [reply capstone-frontend/src/styles/Groups.css]: Will do!

File: capstone-frontend/src/styles/Groups.css (hunk 2 above)
Comment (steven-solar) [reply capstone-frontend/src/styles/Groups.css]: Fixed!

**Review by MauroGuerrero20 — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/MyClubs.jsx [hunk 4]
```diff
@@ -28,7 +28,7 @@ class MyClubs extends Component {
           alert(err); 
         });
     
-    let allClubs = memberClubs.concat(ownerClubs.filter((item) => memberClubs.indexOf(item) < 0));
+    let allClubs = memberClubs.concat(ownerClubs.filter((item) => memberClubs.indexOf(item) < 0))
```
Comment (VivekShankar) [inline capstone-frontend/src/components/MyClubs.jsx]: nit: I think you may have accidentally deleted the semicolon here?

**Review by VivekShankar — APPROVED** [review]
LGTM - the comments features looks great!

File: capstone-frontend/src/components/MyClubs.jsx (hunk 4 above)
Comment (steven-solar) [reply capstone-frontend/src/components/MyClubs.jsx]: I did! Thanks for catching that.

**Pull request merged**